### PR TITLE
APPTS 9.0.1-9.0.5 release notes

### PR DESCRIPTION
**Pull request by pkclark**
Thank you for your contribution to the Platform-Management-Docs repo.

## Describe the changes

Enter a brief description of your changes here to communicate to the maintainers what you changed and why.

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [ ] Does not have conflicts with the base branch
* [ ] Is clear and complete
* [ ] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [ ] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [ ] Does not contain typos, grammatical errors, etc.
* [ ] Does not expose any sensitive information
* [ ] Passes the Markdown linter rules (automated via CI check)
* [ ] Does not contain broken links

## Checklist for mergers

_This section is to be filled out by project maintainers only._

Before merging this PR, please make sure:

* [ ] You have applied a production label if this needs a manual push to Zoomin
* [ ] You have added it to the correct GitHub project
* [ ] You have added a new collection to Netlify CMS config (static/admin/config.js) if required (new doc sections)
* [ ] You have updated the Zoomin classification file if required (new folder of topics>

### CODE REVIEW

File: content/en/docs/release_notes/09_00_02_relnotes.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+Hide_readingtime: true
+---
+
+## Platform Management 9.0.2 - 15 December 2021
+
+Platform Management 9.0.2 is a patch release which includes one improvement and two fixed issues.
+
+## Improvement
+
+* _Usage Entry_ view no longer shows fields for metric for which the org is not entitled (zero quota value).
```
Comment (sallylanda6) [inline content/en/docs/release_notes/09_00_02_relnotes.md:16]:
```
I'm having a hard time with this one at the part of "for metric for" but I think it should be "for metric to" (to which the org is not entitled). 
* _Usage Entry_ view no longer shows fields for metric for which the org is not entitled (zero quota value).
```

**Review by sallylanda6 — CHANGES_REQUESTED** [review]
There are some minor edits and a suggested change.

File: content/en/docs/release_notes/09_00_02_relnotes.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+linkTitle: Platform Management 9.0.2 release notes
+description: Improvements and bug fixes for the release.
+weight: 92
+date: 2021-12-15
+Hide_readingtime: true
+---
+
+## Platform Management 9.0.2 - 15 December 2021
+
+Platform Management 9.0.2 is a patch release which includes one improvement and two fixed issues.
```
[the remark this answers is not in the document]
Comment (pkclark) [reply content/en/docs/release_notes/09_00_02_relnotes.md:12]: I can't count 🙄

File: content/en/docs/release_notes/09_00_02_relnotes.md (hunk 1 above)
Comment (pkclark) [reply content/en/docs/release_notes/09_00_02_relnotes.md:16]: Improved this verbiage.

File: content/en/docs/release_notes/09_00_05_relnotes.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+title: Platform Management 9.0.5 release notes
+linkTitle: Platform Management 9.0.5 release notes
+description: Improvement and bug fix for the release.
+weight: 89
+date: 2022-01-06
+Hide_readingtime: true
+---
+
+## Platform Management 9.0.5 - 6 Janurary 2022
```
[the remark this answers is not in the document]
Comment (pkclark) [reply content/en/docs/release_notes/09_00_05_relnotes.md]: Fixed

File: content/en/docs/release_notes/09_00_04_relnotes.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+linkTitle: Platform Management 9.0.4 release notes
+description: Improvement and bug fix for the release.
+weight: 90
+date: 2021-12-17
+Hide_readingtime: true
+---
+
+## Platform Management 9.0.4 - 17 December 2021
+
+Platform Management 9.0.4 is a patch release which includes one improvement and two fixed issues.
```
[the remark this answers is not in the document]
Comment (pkclark) [reply content/en/docs/release_notes/09_00_04_relnotes.md]: Fixed

File: content/en/docs/release_notes/09_00_02_relnotes.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (pkclark) [reply content/en/docs/release_notes/09_00_02_relnotes.md]: Fixed

**Comment by pkclark** [comment]
@sallylanda6 your comments should all be addressed.

**Pull request merged**